apps/asap-server/src/handlers/webhooks/webhook-run-migrations.ts

```diff
@@ -0,0 +1,257 @@
+import { Query, RestMigration, Squidex } from '@asap-hub/squidex';
+import { isBoom } from '@hapi/boom';
+import { Handler } from 'aws-lambda';
+import { promises as fsPromise } from 'fs';
+import { Logger } from 'pino';
+import { migrationDir } from '../../config';
+import pinoLogger from '../../utils/logger';
```

Comment:
```suggestion
import logger from '../../utils/logger';
```

Reply:
If you agree, don't forget to change the references bellow, too

Reply:
I just saw now, there is another `logger` inside the function scope.

Reply:
yeah